### ISSUE TITLE
powerline_npm.lua adds extra λ

### ISSUE THREAD

**Issue by dmwyatt** [AmrEldib/cmder-powerline-prompt#55]
https://i.imgur.com/CDrRkww.png

If I am in a folder that is actually an npm package, that extra lambda disappears and everything works correctly.

I confirmed that this was likely due to `powerline_npm.lua` by removing it from my config directory and restarting cmder.

I can't quite figure this one out myself...

**Comment by CollinChaffin** [comment]
I have confirmed it is actually the new version of powerline_hg.lua causing this.  I have not had the time to dig into it yet to determine why but a very quick diff shows all the refactoring including an init() that I believe is tacking on the prompt regardless of whether it is even that type.  Rolling back to the prior version of powerline_hg.lua is obviously not the long-term solution since it needed the rewrite, but as a short-term fix if you are not worried so much about mercurial.

**Comment by dmwyatt** [comment]
Yeah, if I remove `powerline_hg.lua` it fixes it.  Odd, that removing `powerline_npm.lua` also fixes it...
